x-pack/plugins/security_solution/public/management/pages/trusted_apps/view/components/create_trusted_app_form.test.tsx

```diff
@@ -411,5 +415,48 @@ describe('When using the Trusted App Form', () => {
         },
       });
     });
+
+    it('should not validate form to true if name input is empty', () => {
+      const props = {
+        name: 'some name',
+        description: '',
+        effectScope: {
+          type: 'global',
+        },
+        os: OperatingSystem.WINDOWS,
+        entries: [
+          { field: ConditionEntryField.PATH, operator: 'included', type: 'wildcard', value: 'x' },
+        ],
+      } as NewTrustedApp;
+
+      formProps.trustedApp = props;
+      render();
+
+      formProps.trustedApp = {
+        ...props,
+        name: '',
+      };
+      rerender();
+
+      expect(getAllValidationErrors()).toHaveLength(0);
+      expect(getAllValidationWarnings()).toHaveLength(1);
+      expect(formProps.onChange).toHaveBeenLastCalledWith({
```

Comment:
Why is this called when the name is empty?

Reply:
`onChange` is called when props change isn't it?

Reply:
oh right, I was thinking about the redux action. That's ok then

Reply:
😄